src/index.js

```diff
@@ -0,0 +1,38 @@
+import './style.css';
+
+const todos = [
+  {
+    _id: 1,
+    title: 'Work',
+    description: 'You have to work to live?',
+    completed: false,
+  },
+  {
+    _id: 2,
+    title: 'Eat',
+    description: 'You have to eat to live?',
+    completed: false,
+  },
+  {
+    _id: 3,
+    title: 'Sleep',
+    description: 'You have to sleep to live?',
+    completed: false,
+  },
```

Comment:
- Hey, according to the project [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md), the object should only contain `index, description, and completed`. please, update your object to have the above properties and remove properties not listed in the requirement.

```
 {
    index: 1,
    description: 'An example of your description',
    completed: false,
  }
```